main.go

```diff
@@ -2,55 +2,43 @@ package main
 
 import (
 	"fmt"
+	"github.com/chainHero/heroes-service/blockchain"
+	"github.com/chainHero/heroes-service/web"
+	"github.com/chainHero/heroes-service/web/controllers"
 	"os"
-	"runtime"
-	"path/filepath"
-	"github.com/chainhero/heroes-service/blockchain"
-	"github.com/chainhero/heroes-service/web"
-	"github.com/chainhero/heroes-service/web/controllers"
 )
 
-// Fix empty GOPATH with golang 1.8 (see https://github.com/golang/go/blob/1363eeba6589fca217e155c829b2a7c00bc32a92/src/go/build/build.go#L260-L277)
-func defaultGOPATH() string {
-	env := "HOME"
-	if runtime.GOOS == "windows" {
-		env = "USERPROFILE"
-	} else if runtime.GOOS == "plan9" {
-		env = "home"
-	}
-	if home := os.Getenv(env); home != "" {
-		def := filepath.Join(home, "go")
-		if filepath.Clean(def) == filepath.Clean(runtime.GOROOT()) {
-			// Don't set the default GOPATH to GOROOT,
-			// as that will trigger warnings from the go tool.
-			return ""
-		}
-		return def
-	}
-	return ""
-}
-
 func main() {
-	// Setup correctly the GOPATH in the environment
-	if goPath := os.Getenv("GOPATH"); goPath == "" {
-		os.Setenv("GOPATH", defaultGOPATH())
+	// Definition of the Fabric SDK properties
+	fSetup := blockchain.FabricSetup{
+		// Channel parameters
+		ChannelID:     "chainhero",
+		ChannelConfig: "" + os.Getenv("GOPATH") + "/src/github.com/chainHero/heroes-service/fixtures/artifacts/",
```

Comment:
Set the full path here and remove the prefix `"" + ` (no required): `os.Getenv("GOPATH") + "/src/github.com/chainHero/heroes-service/fixtures/artifacts/chainhero.channel.tx"`

Reply:
@MaximeAubanel

Reply:
Done